src/oneNoteApi.d.ts

```diff
@@ -137,6 +147,13 @@ declare namespace OneNoteApi {
 		pages: Page[];
 	}
 
+	export interface Revision {
+		target: string;
+		action: string;
```

Comment:
Let's type this to "append" | "insert" | "prepend" | "replace", or even better, an enum (https://dev.onenote.com/docs#/reference/patch-pages)